Game/Source/GDKTestGyms/Public/Task_DelaySimulated.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameplayTask.h"
+
+#include "Task_DelaySimulated.generated.h"
+
+UCLASS()
+class GDKTESTGYMS_API UTask_DelaySimulated : public UGameplayTask
+{
+	GENERATED_BODY()
+
+public:
+	UTask_DelaySimulated(const FObjectInitializer& ObjectInitializer);
+
+	virtual void Activate() override;
+
+	virtual void InitSimulatedTask(UGameplayTasksComponent& InGameplayTasksComponent) override;
+
+	UFUNCTION(BlueprintCallable, Category = "SimulatedGameplayTask", meta = (BlueprintInternalUseOnly = "TRUE"))
+	static UTask_DelaySimulated* TaskDelaySimulated(TScriptInterface<IGameplayTaskOwnerInterface> TaskOwner, const uint8 Priority = 192);
```

Comment:
Magic number?

Reply:
Wanted to put `FGameplayTasks::DefaultPriority` but UHT doesn't know what it is and freaks out, so this has to be a literal.

Reply:
Cool - can you add a comment about what it refers to?